### DISCUSSION TITLE
QSelect - prevent context menu and display custom popup

### DISCUSSION THREAD

**Discussion by lukejohnt** [quasarframework/quasar#16001] [General - Components / Directives / etc] [answered]
I am trying to prevent the context menu from displaying on a QSelect control when the user right clicks the mouse and have it display a custom popup. I have it working on other control like QInput using the @contextmenu.prevent="onFieldClick", but can't get it to work on the QSelect.

Thanks

**Comment by pdanpdan** [comment, marked as answer]
You can try wrapping qselect in a div and use capture on `@contextmenu`

**Reply by lukejohnt** [reply]
Thanks, that works :)
